lib/extractors/adoption-type.js

```diff
@@ -0,0 +1,7 @@
+import {RDF, LBLOD_BESLUIT} from '../../util/namespaces';
+import {default as defaultExtract} from './default-extractor';
+
+export default function extract({graph, base, tmp}) {
+  const types = defaultExtract({graph, base, find: LBLOD_BESLUIT('AdoptionType')});
```

Comment:
I think you still would need a replaceWith since now the resulting predicate is weird, e..g
```"http://data.lblod.info/form-data/863cadf0-f10b-11eb-a08f-83d03a536f3c","http://lblod.data.gift/vocabularies/besluit/MunicipalRoadProcedureType","http://lblod.data.gift/concepts/82813418-356b-49a6-be9e-c5ef1b7a70bc"
"http://data.lblod.info/form-data/863cadf0-f10b-11eb-a08f-83d03a536f3c","http://lblod.data.gift/vocabularies/besluit/AdoptionType","http://lblod.data.gift/concepts/28da07ad-4a25-460e-b1be-92bd4b7b8927"

https://github.com/lblod/toezicht-flattened-form-data-generator/blob/master/lib/extractors/tax-type.js#L5
```

Reply:
You mean weird because of the capital A ? Also found it strange but that's how it's defined in the form definition ( https://github.com/lblod/app-digitaal-loket/blob/master/config/semantic-forms/20210706102126-forms.ttl#L85 ) . So I should put a custom predicate for it ? `lblodBesluit:adoptionType` ?

Reply:
yeah, you're right. It was wrong at the source data. Let's make an issue to fix this later.